server/api/forges/[forge_id]/index.post.ts

```diff
@@ -0,0 +1,26 @@
+import { forgeSchema } from '../../../schemas';
+import { eq, and } from 'drizzle-orm';
+
+export default defineEventHandler(async (event) => {
+  const user = await requireUser(event);
+
+  const forgeData = await readBody(event);
+
+  const _forgeId = event.context.params?.forge_id;
+  if (!_forgeId) {
+    throw createError({
+      message: 'Cannot create forge with a forge_id',
+      status: 400,
+    });
+  }
+  const forgeId = parseInt(_forgeId, 10);
+
+  const forge = await db
+    .update(forgeSchema)
```

Comment:
@anbraten I think file should be named index.put.ts or index.patch.ts as and when we are updating it the method type should be 'put' or 'patch' respectively.

Reply:
good catch will adjust